docs/reference/transactions.md

```diff
@@ -2,6 +2,8 @@ title: Transaction Reference
 
 Each table below specifies the **field name**, whether it is optional or required, its **type** within the protocol code (note that SDKs input types for these fields may differ), the **codec**, which is the name of the field when viewed within a transaction, and a **description** of the field. 
 
+**Important**: When signing, the fields have to be sorted alphabetically to match the Algorand canonical encoding.
```

Comment:
Good addition :) 

Minor fixup: Please replace "**Important:**" with "!!! Important" then a new line. Make sure this second line is tabbed over and include the message. Ensure there is a blank line above and below this section. This MD formatting delivers a pretty UI box.

Reply:
Thanks for the suggestions! !!! Important isn't working?
```
!!! Important

    When signing, the fields have to be sorted alphabetically to match the Algorand canonical encoding.
```
!!! Important

    When signing, the fields have to be sorted alphabetically to match the Algorand canonical encoding.